apps/backend/src/users/users.controller.ts

```diff
@@ -7,7 +13,10 @@ export class UsersController {
   constructor(private readonly usersService: UsersService) {}
 
   @Get()
-  getAllUsers() {
-    return this.usersService.findAll();
+  getAllMembers(
+    @Query('getAllMembers', new DefaultValuePipe(false), ParseBoolPipe)
```

Comment:
:question: Does `@Query('getAllMembers', ParseBoolPipe) getAllMembers: boolean` also work here? (Also just a side note that the `getAllMembers` param should actually be called `allMembers`)

Reply:
I believe that breaks the endpoint when you don't pass in any query parameters (i.e., `localhost:3000/api/users/`). This is because `ParseBoolPipe` expects a non-null, non-undefined value as inputs. Since we want the default value to be false when no query parameter is passed, as per the ticket, we decided to use the `DefaultValuePipe`. We can also change the param name to `allMembers`

Reply:
Ahh yep, makes sense!